### DISCUSSION TITLE
Open Source Contributions

### DISCUSSION THREAD

**Discussion by ayushsisodia7** [community/community#58740] [GitHub Education]
### Hi everyone,

I want to start with open source contributions. Please suggest/guide me how I can contribute.

**Comment by the-officialjosh** [comment]
check out this tutorial by  [Free Code Camp](https://www.youtube.com/watch?v=yzeVMecydCE&t=300s)

**Reply by ayushsisodia7** [reply]
Will check it out
Thanks

**Comment by leonden** [comment]
There is a great [repository](https://github.com/firstcontributions/first-contributions) which teaches you exactly that.
You can simply start by picking a project you would like to contribute, best suited ist software that you already use. Get to know the rules of contribution in that particular project. Then go to the issues on the repository, clone it and create a new branch with the name of the problem. You can change the code accordingly and open a pull request.
Hope that helps 😄

**Reply by ayushsisodia7** [reply]
Will check it out
Thanks

**Reply by leonden** [reply]
If this solves your request, please consider marking my post as the answer to close the discussion :)
